lib/node_modules/@stdlib/math/base/special/besselj1/benchmark/r/benchmark.R

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env Rscript
+
+# Set the precision to 16 digits:
+options( digits = 16 );
+
+#' Run benchmarks.
+#'
+#' @examples
+#' main();
+main <- function() {
+	# Define benchmark parameters:
+	name <- "besselj1";
+	iterations <- 1000000L;
+	repeats <- 3;
+
+	#' Print the TAP version.
+	#'
+	#' @examples
+	#' print_version();
+	print_version <- function() {
+		cat( "TAP version 13\n" );
+	}
+
+	#' Print the TAP summary.
+	#'
+	#' @param total Total number of tests.
+	#' @param passing Total number of passing tests.
+	#'
+	#' @examples
+	#' print_summary( 3, 3 );
+	print_summary <- function( total, passing ) {
+		cat( "#\n" );
```

Comment:
Looks like we still have indentation issues in this file.

Reply:
I think I got it. I had to set `:noet` (no expand tab) and mess with tabstop in vim…

Reply:
Oh nice. At some point (no rush), if you want to write a Vim guide (https://github.com/stdlib-js/stdlib/tree/develop/docs/editors) related to the project (linters and whatnot), that would be fab. :)